exec/cnex/lib/datetime.c

```diff
@@ -0,0 +1,57 @@
+#ifdef _MSC_VER
+#define _CRT_SECURE_NO_WARNINGS
+#define timegm _mkgmtime
+#else
+#define _DEFAULT_SOURCE    // For timegm on linux
+#endif
+#include "datetime.h"
+
+#include <time.h>
+
+#include "array.h"
+#include "cell.h"
+#include "exec.h"
+#include "number.h"
+#include "stack.h"
+
+
+
+void datetime_gmtime(TExecutor *exec)
+{
+    Number t = top(exec->stack)->number;
+
+    time_t x = number_to_sint32(t);
```

Comment:
32 looks wrong here. Change it to 64, I've just made the corresponding change in datetime.cpp.

Reply:
Done.